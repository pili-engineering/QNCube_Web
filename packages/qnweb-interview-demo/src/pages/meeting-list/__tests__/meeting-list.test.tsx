import React from 'react';
import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';

import MeetingList from '..';

describe('MeetingList', () => {
  it('first render', () => {
    const component = render(<MeetingList/>);
    expect(component).toMatchSnapshot();
  });
});
