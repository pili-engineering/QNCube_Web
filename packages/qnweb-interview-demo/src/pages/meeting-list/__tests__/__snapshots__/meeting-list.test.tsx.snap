// Vitest Snapshot v1

exports[`MeetingList > first render 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="common-header"
      >
        <div
          class="header"
        >
          <div
            class="header-logo"
          >
            <img
              alt=""
              src="/src/components/common-header/logo.png"
            />
          </div>
          <div
            class="header-user"
          >
            <span />
            <button
              class="ant-btn ant-btn-primary"
              type="button"
            >
              <span>
                新建面试
              </span>
            </button>
          </div>
        </div>
        <div
          class="_container_1ur1w_1"
        >
          <div
            class="ant-table-wrapper interview-table"
          >
            <div
              class="ant-spin-nested-loading"
            >
              <div
                class="ant-spin-container"
              >
                <div
                  class="ant-table ant-table-empty"
                >
                  <div
                    class="ant-table-container"
                  >
                    <div
                      class="ant-table-content"
                    >
                      <table
                        style="table-layout: auto;"
                      >
                        <colgroup />
                        <thead
                          class="ant-table-thead"
                        >
                          <tr>
                            <th
                              class="ant-table-cell"
                              style="text-align: center;"
                            >
                              面试标题
                            </th>
                            <th
                              class="ant-table-cell"
                              style="text-align: center;"
                            >
                              应聘者姓名
                            </th>
                            <th
                              class="ant-table-cell"
                              style="text-align: center;"
                            >
                              应聘者手机号
                            </th>
                            <th
                              class="ant-table-cell"
                              style="text-align: center;"
                            >
                              是否录制
                            </th>
                            <th
                              class="ant-table-cell"
                              style="text-align: center;"
                            >
                              面试状态
                            </th>
                            <th
                              class="ant-table-cell"
                              style="text-align: center;"
                            >
                              公司/职位信息
                            </th>
                            <th
                              class="ant-table-cell"
                              style="text-align: center;"
                            >
                              时间
                            </th>
                            <th
                              class="ant-table-cell"
                              style="text-align: center;"
                            >
                              操作
                            </th>
                          </tr>
                        </thead>
                        <tbody
                          class="ant-table-tbody"
                        >
                          <tr
                            class="ant-table-placeholder"
                          >
                            <td
                              class="ant-table-cell"
                              colspan="8"
                            >
                              <div
                                class="ant-empty ant-empty-normal"
                              >
                                <div
                                  class="ant-empty-image"
                                >
                                  <svg
                                    class="ant-empty-img-simple"
                                    height="41"
                                    viewBox="0 0 64 41"
                                    width="64"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <g
                                      fill="none"
                                      fill-rule="evenodd"
                                      transform="translate(0 1)"
                                    >
                                      <ellipse
                                        class="ant-empty-img-simple-ellipse"
                                        cx="32"
                                        cy="33"
                                        rx="32"
                                        ry="7"
                                      />
                                      <g
                                        class="ant-empty-img-simple-g"
                                        fill-rule="nonzero"
                                      >
                                        <path
                                          d="M55 12.76L44.854 1.258C44.367.474 43.656 0 42.907 0H21.093c-.749 0-1.46.474-1.947 1.257L9 12.761V22h46v-9.24z"
                                        />
                                        <path
                                          class="ant-empty-img-simple-path"
                                          d="M41.613 15.931c0-1.605.994-2.93 2.227-2.931H55v18.137C55 33.26 53.68 35 52.05 35h-40.1C10.32 35 9 33.259 9 31.137V13h11.16c1.233 0 2.227 1.323 2.227 2.928v.022c0 1.605 1.005 2.901 2.237 2.901h14.752c1.232 0 2.237-1.308 2.237-2.913v-.007z"
                                        />
                                      </g>
                                    </g>
                                  </svg>
                                </div>
                                <div
                                  class="ant-empty-description"
                                >
                                  No Data
                                </div>
                              </div>
                            </td>
                          </tr>
                        </tbody>
                      </table>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="common-header"
    >
      <div
        class="header"
      >
        <div
          class="header-logo"
        >
          <img
            alt=""
            src="/src/components/common-header/logo.png"
          />
        </div>
        <div
          class="header-user"
        >
          <span />
          <button
            class="ant-btn ant-btn-primary"
            type="button"
          >
            <span>
              新建面试
            </span>
          </button>
        </div>
      </div>
      <div
        class="_container_1ur1w_1"
      >
        <div
          class="ant-table-wrapper interview-table"
        >
          <div
            class="ant-spin-nested-loading"
          >
            <div
              class="ant-spin-container"
            >
              <div
                class="ant-table ant-table-empty"
              >
                <div
                  class="ant-table-container"
                >
                  <div
                    class="ant-table-content"
                  >
                    <table
                      style="table-layout: auto;"
                    >
                      <colgroup />
                      <thead
                        class="ant-table-thead"
                      >
                        <tr>
                          <th
                            class="ant-table-cell"
                            style="text-align: center;"
                          >
                            面试标题
                          </th>
                          <th
                            class="ant-table-cell"
                            style="text-align: center;"
                          >
                            应聘者姓名
                          </th>
                          <th
                            class="ant-table-cell"
                            style="text-align: center;"
                          >
                            应聘者手机号
                          </th>
                          <th
                            class="ant-table-cell"
                            style="text-align: center;"
                          >
                            是否录制
                          </th>
                          <th
                            class="ant-table-cell"
                            style="text-align: center;"
                          >
                            面试状态
                          </th>
                          <th
                            class="ant-table-cell"
                            style="text-align: center;"
                          >
                            公司/职位信息
                          </th>
                          <th
                            class="ant-table-cell"
                            style="text-align: center;"
                          >
                            时间
                          </th>
                          <th
                            class="ant-table-cell"
                            style="text-align: center;"
                          >
                            操作
                          </th>
                        </tr>
                      </thead>
                      <tbody
                        class="ant-table-tbody"
                      >
                        <tr
                          class="ant-table-placeholder"
                        >
                          <td
                            class="ant-table-cell"
                            colspan="8"
                          >
                            <div
                              class="ant-empty ant-empty-normal"
                            >
                              <div
                                class="ant-empty-image"
                              >
                                <svg
                                  class="ant-empty-img-simple"
                                  height="41"
                                  viewBox="0 0 64 41"
                                  width="64"
                                  xmlns="http://www.w3.org/2000/svg"
                                >
                                  <g
                                    fill="none"
                                    fill-rule="evenodd"
                                    transform="translate(0 1)"
                                  >
                                    <ellipse
                                      class="ant-empty-img-simple-ellipse"
                                      cx="32"
                                      cy="33"
                                      rx="32"
                                      ry="7"
                                    />
                                    <g
                                      class="ant-empty-img-simple-g"
                                      fill-rule="nonzero"
                                    >
                                      <path
                                        d="M55 12.76L44.854 1.258C44.367.474 43.656 0 42.907 0H21.093c-.749 0-1.46.474-1.947 1.257L9 12.761V22h46v-9.24z"
                                      />
                                      <path
                                        class="ant-empty-img-simple-path"
                                        d="M41.613 15.931c0-1.605.994-2.93 2.227-2.931H55v18.137C55 33.26 53.68 35 52.05 35h-40.1C10.32 35 9 33.259 9 31.137V13h11.16c1.233 0 2.227 1.323 2.227 2.928v.022c0 1.605 1.005 2.901 2.237 2.901h14.752c1.232 0 2.237-1.308 2.237-2.913v-.007z"
                                      />
                                    </g>
                                  </g>
                                </svg>
                              </div>
                              <div
                                class="ant-empty-description"
                              >
                                No Data
                              </div>
                            </div>
                          </td>
                        </tr>
                      </tbody>
                    </table>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
